src/client.ts

```diff
@@ -1,9 +1,12 @@
-// import {bootstrap} from 'angular2/platform/browser';
-import {bootstrap} from 'angular2-universal-preview';
+//import 'angular2-universal-preview/polyfills';
```

Comment:
we don't need `angular2-universal-preview/polyfills` since the polyfills are in the `index.html`